### PR TITLE
Return query statistics

### PR DESCRIPTION
**Pull request by thenineteen**
The three most crucial functions are updated to return extra variables as per issue #83. These include numbers of studies, patients, localising and lateralising datapoints. 

The three files are:
`QUERY_SEMIOLOGY`
`QUERY_LATERALISATION`
`MEGA_ANALYSIS`

This will break the code as we will get the following exception everywhere:

`Exception has occurred: ValueError
too many values to unpack`

### CODE REVIEW

**Comment by thenineteen** [comment]
I've fixed all low level references to the three functions as above.

@fepegar can you please do the same for the top level references (in your `semiology.py` file which uses lower case functions of the same names)

File: mega_analysis/semiology.py [hunk 1]
```diff
@@ -121,7 +122,7 @@ def remove_exclusions(
             include_cortical_stimulation: bool,
             include_et_topology_ez: bool,
             include_spontaneous_semiology: bool,
-            ) -> pd.DataFrame:
+    ) -> pd.DataFrame:
```
Comment (fepegar) [inline mega_analysis/semiology.py:125]: Please stop de-indentating these brackets.

Comment (thenineteen) [reply mega_analysis/semiology.py:125]: this is automatic pep8 and linting which I enabled because of issue #50

**Comment by thenineteen** [comment]
> I don't know what you mean by "fix references". Can you please elaborate?

as in when we used `a = QUERY_SEMIOLOGY()`, now the function returns many more variables, so all references to the function were updated e.g. `a, b, c = QUERY_SEMIOLOGY()`

**Comment by thenineteen** [comment]
there is the following error arising from top level query_lateralisation

`
TypeError: cannot unpack non-iterable NoneType object
mega_analysis/semiology.py:175: TypeError
`

**Comment by fepegar** [comment]
> as in when we used a = QUERY_SEMIOLOGY(), now the function returns many more variables, so all references to the function were updated e.g. a, b, c = QUERY_SEMIOLOGY()

You seem to have gone through `semiology` for that. That error happens because `QUERY_LATERALISATION` is returning `None`.

**Comment by thenineteen** [comment]
> > as in when we used a = QUERY_SEMIOLOGY(), now the function returns many more variables, so all references to the function were updated e.g. a, b, c = QUERY_SEMIOLOGY()
> 
> You seem to have gone through `semiology` for that. That error happens because `QUERY_LATERALISATION` is returning `None`.

I'm not sure I follow. Yes I also updated the calls to the QUERY_... in `semiology` but this shouldn't break it... when you say returning None, as in, all the returns are None? or just the first (original)?

**Comment by thenineteen** [comment]
Oh I understand now. The code breaks at some points and return None without returning those statistics. 

Need to adapt what happens when it returns None. Can it instead return `None, a, b, c`?

File: mega_analysis/semiology.py (hunk 1 above)
Comment (fepegar) [reply mega_analysis/semiology.py:125]: I know, but this is not explicitly specified in PEP8 and you can just not stage these changes for the commit. The point is to make the code more readable, and this doesn't as indentation is lost.

**Comment by fepegar** [comment]
> Need to adapt what happens when it returns None. Can it instead return None, a, b, c?

I think that would work. But please in the future let me adapt the `semiology` module to any changes, as it's what I use as link between `mega_analysis` and Slicer.

**Comment by thenineteen** [comment]
so there are 4 file merge conflicts, 3 I fixed, but then there is semiology.py which you need to resolve the merge conflicts. I couldn't figure out a way to save my changes on GitHub and commit them while leaving semioloy.py for you to resolve?

**Comment by thenineteen** [comment]
I don't understand, are there changes you've requested you wanted me to review?

**Comment by thenineteen** [comment]
any update on this yet?

**Comment by fepegar** [comment]
There are still a lot of conflicts. I suggest you rebase from master and solve them before moving forward. The first thing you can do is squashing all the commits in this PR (also using rebase).

**Comment by thenineteen** [comment]
ok this is done now, passes all tests

**Pull request merged**